util/chplenv/chpl_llvm.py

```diff
@@ -327,20 +332,21 @@ def validate_llvm_config():
                   .format(llvm_config, config_error))
 
     if llvm_val == 'system':
-      bindir = get_system_llvm_config_bindir()
-      if not (bindir and os.path.isdir(bindir)):
-          error("llvm-config command {0} provides missing bin dir {1}"
-                .format(llvm_config, bindir))
-      clang_c = get_llvm_clang('c')[0]
-      clang_cxx = get_llvm_clang('c++')[0]
-      if not os.path.exists(clang_c):
-          error("Missing clang command at {0}".format(clang_c))
-      if not os.path.exists(clang_cxx):
-          error("Missing clang++ command at {0}".format(clang_cxx))
-
-      (noPackageErrors, package_err) = check_llvm_packages(llvm_config)
-      if not noPackageErrors:
-        error(package_err)
+        bindir = get_system_llvm_config_bindir()
+        if "/nix/store" not in bindir:
```

Comment:
same point / question here about maybe it being better to use `not bindir.startswith("/nix/store")`